awx/api/serializers.py

```diff
@@ -2051,7 +2052,7 @@ class InventorySourceSerializer(UnifiedJobTemplateSerializer, InventorySourceOpt
 
     class Meta:
         model = InventorySource
-        fields = ('*', 'name', 'inventory', 'update_on_launch', 'update_cache_timeout', 'source_project') + (
+        fields = ('*', 'name', 'inventory', 'update_on_launch', 'update_cache_timeout', 'source_project', 'scm_branch') + (
```

Comment:
This field was already added to `InventorySourceOptionsSerializer`, a parent class of this, so I don't think it should be added again here.

Reply:
fixed